### PR TITLE
create topic disk allocate strategy per broker

### PR DESCRIPTION
**Pull request by seayoun**
*More detailed description of your change,
if necessary. The PR title and PR message become
the squashed commit message, so use a separate
comment to ping reviewers.*

*Summary of testing strategy (including rationale)
for the feature or bug fix. Unit and/or integration
tests are expected for any behaviour change and
system tests should be considered for larger changes.*

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

**Comment by kaushiksrinivas** [comment]
retest this please.

File: tools/src/main/java/org/apache/kafka/tools/ProducerPerformance.java [hunk 1]
```diff
@@ -126,7 +126,7 @@ public static void main(String[] args) throws Exception {
 
             int currentTransactionSize = 0;
             long transactionStartTime = 0;
-            for (int i = 0; i < numRecords; i++) {
+            for (long i = 0; i < numRecords; i++) {
```
Comment (omkreddy) [inline tools/src/main/java/org/apache/kafka/tools/ProducerPerformance.java:129]: I have already merged this change as part of #5956

**Comment by omkreddy** [comment]
@seayoun  Request you to create a JIRA and describe the requirement/change. This will be useful to understand/track the changes.
https://issues.apache.org/jira/projects/KAFKA/

**Comment by seayoun** [comment]
ok, thank you

**Comment by seayoun** [comment]
> @seayoun Request you to create a JIRA and describe the requirement/change. This will be useful to understand/track the changes.
> https://issues.apache.org/jira/projects/KAFKA/

I submit an issue on jira https://issues.apache.org/jira/browse/KAFKA-7690 , I am Looking forward to your suggestions.

**Comment by seayoun** [comment]
> retest this please.

discuss here https://issues.apache.org/jira/browse/KAFKA-7690

**Pull request closed without merging**